releasenotes/notes/redirect-dns-iptables.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: feature
+area: networking
+releaseNotes:
+- |
+  **Added** Allow specifying dns capture via CLI in istio-iptables script.
```

No capitalization of word after **..**

Maybe something like:
```suggestion
  **Added** flag to enable capture of dns traffic to the istio-iptables script.
```